.devcontainer/devcontainer.json

```diff
@@ -37,5 +37,9 @@
         "tamasfe.even-better-toml"
       ]
     }
+  },
+  "remoteEnv": {
+    // Attempt to fix Pylance crashes that seem to be caused by running out of memory.
+    "NODE_OPTIONS": "--max-old-space-size=8192"
```

Comment:
I'm confused by this one? But if it works... 👍

Reply:
Ah, sorry, forgot to include a reference to what led me to this:

- https://github.com/microsoft/pylance-release/blob/main/TROUBLESHOOTING.md#increase-memory-limit-for-vs-code-remote-only

I don't totally understand how it works behind the scenes, but it definitely seemed to reduced the occurrence of crashes for me.